library/pom.xml

```diff
@@ -315,7 +315,17 @@
       <artifactId>joda-time</artifactId>
       <version>2.9.1</version>
     </dependency>
-
+    <dependency>
```

Comment:
I agree, but this case requires it. This is an enhancement to existing operator in malhar-library